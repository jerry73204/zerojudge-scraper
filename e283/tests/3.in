3
0 1 1 1
1 0 0 0
0 1 1 1
16
0 1 0 1
1 1 0 0
1 1 0 1
1 0 0 0
1 1 0 0
1 1 0 0
0 1 1 1
1 1 0 0
0 0 1 0
0 1 1 1
0 0 1 0
0 1 0 1
0 1 0 1
1 1 0 0
0 1 1 1
0 0 1 0
4
1 1 0 1
0 1 1 1
1 1 0 1
1 0 0 0
12
1 1 0 0
0 1 1 1
1 1 0 0
1 0 0 0
0 0 1 0
0 0 1 0
1 0 0 0
0 0 1 0
1 1 0 1
0 1 0 1
1 0 0 0
0 1 1 1
20
0 1 0 1
1 1 0 1
1 1 0 0
0 1 1 1
0 0 1 0
1 1 0 1
0 1 1 1
0 0 1 0
1 1 0 1
1 0 0 0
0 0 1 0
0 1 0 1
0 1 0 1
0 0 1 0
1 0 0 0
1 1 0 1
1 1 0 0
1 0 0 0
1 1 0 0
0 1 1 1
1
1 1 0 0
5
0 0 1 0
1 1 0 1
0 1 1 1
0 1 0 1
0 0 1 0
14
0 0 1 0
0 1 0 1
0 1 1 1
0 1 0 1
1 1 0 0
0 1 0 1
0 1 0 1
0 1 0 1
1 1 0 0
1 1 0 1
0 1 1 1
0 0 1 0
0 1 0 1
1 0 0 0
8
1 0 0 0
0 1 1 1
0 1 0 1
0 1 0 1
1 0 0 0
0 1 1 1
0 1 0 1
1 1 0 1
11
1 1 0 1
0 1 0 1
1 1 0 0
0 1 0 1
0 1 0 1
0 1 1 1
1 0 0 0
1 0 0 0
0 1 1 1
0 1 1 1
0 1 1 1
10
0 0 1 0
0 1 0 1
0 0 1 0
0 1 1 1
1 1 0 1
0 1 0 1
0 1 0 1
0 0 1 0
0 1 1 1
1 0 0 0
5
0 1 0 1
1 1 0 0
0 1 1 1
1 1 0 0
0 1 1 1
5
1 0 0 0
1 1 0 1
1 0 0 0
1 1 0 1
1 1 0 0
2
0 0 1 0
1 0 0 0
2
0 0 1 0
0 1 0 1
6
1 1 0 1
1 1 0 0
1 1 0 1
0 1 1 1
1 1 0 0
0 0 1 0
14
0 0 1 0
0 1 0 1
1 0 0 0
0 1 0 1
1 1 0 0
1 1 0 1
0 1 1 1
0 1 0 1
0 1 0 1
1 1 0 0
0 1 0 1
1 0 0 0
0 1 1 1
1 1 0 1
10
1 0 0 0
0 0 1 0
0 1 1 1
0 1 0 1
1 1 0 1
1 1 0 1
0 0 1 0
1 0 0 0
0 0 1 0
1 1 0 0
