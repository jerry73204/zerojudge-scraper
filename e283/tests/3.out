BEB
AFDEFFBFCBCAAFBC
DBDE
FBFECCECDAEB
ADFBCDBCDECAACEDFEFB
F
CDBAC
CABAFAAAFDBCAE
EBAAEBAD
DAFAABEEBBB
CACBDAACBE
AFBFB
EDEDF
CE
CA
DFDBFC
CAEAFDBAAFAEBD
ECBADDCECF